host_applications/linux/apps/raspicam/RaspiCommonSettings.c

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright (c) 2018, Raspberry Pi (Trading)
```

Comment:
You need to be consistent and correct with the Copyright notices. In the kernel we use:
{noformat}
Copyright (C) 2018 Raspberry Pi (Trading) Ltd.
{noformat}

Reply:
Doing..

Reply:
You are still missing the '.' off 'Ltd.', but IANAL.